contracts/LifMarketMaker.sol

```diff
@@ -44,9 +44,16 @@ contract LifMarketMaker is Ownable {
 
   bool public funded = false;
 
+  // if the market maker is paused or not
+  bool public paused = false;
+
+  // total amount of blocks that the maret maker was paused
+  uint256 public totalBlocksPaused = 0;
+
+  // the block where the cmarket maker was paused
```

Comment:
"market" has a typo here and in previous comment.
Also, what about changing `blockPaused` to `lastPausedBlock`? Or actually, why don't we change `paused` from bool to uint256 so it represents the block since which it's paused? We get rid of a var that way, we would set it back to 0 when it's not paused. What do you think?

Reply:
I thought in that, but I think keeping that bool makes it more understandable, if you want to know if is paused or not as a dev you will look for a bool variable called `paused` and since its just a bool, its not expensive to keep it. 
About changing the name of `blockPaused` I can change it `pausedBlock` and `totalBlocksPaused` to `totalPausedBlocks`